dash_eee/pages/Home.py

```diff
@@ -0,0 +1,95 @@
+from dash import Dash, html, Input, Output, dcc
+from dash_bootstrap_templates import load_figure_template
+import dash
+import plotly.express as px
+import dash_bootstrap_components as dbc
+import json
+import os
+
+old_path = os.getcwd()
+os.chdir("..")
+
+new_path = os.getcwd() + '/app/Data/users_data.json'
+
+with open(new_path, 'r') as f:
+   data = json.load(f)
+
+os.chdir(old_path)
+
+
+user_list = []
+for i in data:
+    user = f'{i["first_name"]} {i["last_name"]}, user id: {i["user_id"]}'
+    user_list.append(user)
+
+
+image_path='assets/deloton.png'
+dropdowns = html.Div(
+    [
+        dcc.Dropdown(
+            options=user_list,
+            id = 'first-dropdown'
+        )],
+)
+
+card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H4("Card title", className="card-title"),
+                html.P(
+                    "Some quick example text to build on the card title and "
+                    "make up the bulk of the card's content.",
+                    className="card-text",
+                ),
+                dcc.Graph(id='example-graph')
+            ]
+        ),
+    ],
+    style={"width": "29rem"},
+)
+
```

Comment:
there is alot of repeated code here maybe i could create a function that creates the row and allows 3 inputs for the 3 columns per row instead of copying and pasting the same stuff over and over